server/sonar-server/src/main/java/org/sonar/server/es/DefaultIndexSettings.java

```diff
@@ -28,6 +28,9 @@
   /** Maximum length of ngrams. */
   public static final int MAXIMUM_NGRAM_LENGTH = 15;
 
+  /** Pattern, that splits the user search input **/
+  public static final String SEARCH_TERM_TOKENIZER_PATTERN = "[\\s]+";
```

Comment:
This constant is specific to the context of `ComponentIndexSearchFeature` and should move accordingly.